template-only-bin/install-template.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -euo pipefail
+
+echo "Fetch latest version of template-application-nextjs"
+git clone --single-branch --branch main --depth 1 git@github.com:navapbc/template-application-nextjs.git
+
+echo "Copy files from template-application-nextjs"
+cd template-application-nextjs
+cp -r \
```

Comment:
note i didn't copy over README or LICENSE, i feel like projects should do those on their own, i don't see much use for making a template version of those